magento/pwa-studio
# fix: clear url search segment when final filter removed - fixes #1921

# Description

No longer push a new history state containing empty values

## Related Issue
Closes #1921 .

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to any product list
2. Select 1 or more filters & hit click 'apply filters'
3. remove all filters
4. see that the url now only contains the pathname.
